### ISSUE TITLE
Rules incorrectly reordered in output

### ISSUE THREAD

**Issue by ahwayakchih** [ben-eb/postcss-merge-rules#24]
**Fixture:**

```css
.foo { background: #fff; -webkit-background-clip: text; }
.bar { background: #000; -webkit-background-clip: text; }
```

**Expected:**

Rules not merged or something like:

```css
.foo { background: #fff; } .bar { background: #000; } .foo,.bar { -webkit-background-clip: text; }
```

**Actual:**

When using `-webkit-background-clip`, rules are merged, but incorrect order makes background-clip effect disappear from second rule:

```css
.foo { background: #fff; } .foo,.bar { -webkit-background-clip: text; } .bar { background: #000; }
```

After dropping vendor prefix (using `background-clip`), result is correct (although rules are not merged):

```css
.foo { background: #fff; background-clip: text; } .bar { background: #000; background-clip: text; }
```

**Comment by ben-eb** [comment]
Resolved in `2.0.9`. ✨ 

By the way, many thanks for the thorough reporting! 👍

**Comment by ahwayakchih** [comment]
Thanks :).
